casper/order/order_estimator.py

```diff
@@ -0,0 +1,10 @@
+def get_estimate_from_latest_messages(latest_bets):
+    sample_list = list(latest_bets.values())[0].estimate
+    elem_weights = {elem: 0 for elem in sample_list}
```

Comment:
Given that this is a dictionary, you implicitly enforce that there aren't any repeated elements in the list. I think this is fine

The two solutions here are a) move from calling this ordering a list to calling it ordering a set, b) if there multiple of one type of element, it's score is the sum of all it's scores (and you put the two side by side). The second one seems harder to reason about, though (and there might be some reason I'm missing that it doesn't work).

Reply:
right. I've been assuming unique elements in the list. Started thinking of this initially in terms of the validators ordering themselves to then do something else. Maybe validators coming to consensus on the next set of round robins. Casper on casper lol.

hm sum of the two repeat elements would give it mega priority over the others. I'm cool with us just ordering a set for now :P